ui/helpers/constants.js

```diff
@@ -237,7 +237,7 @@ export const thPlatformMap = {
   'android-hw-p5-13-0-android-aarch64-qr': 'Android 13.0 Pixel5 AArch64',
   'android-hw-p5-13-0-arm7-shippable-qr': 'Android 13.0 Pixel5 Arm7 Shippable',
   'android-hw-p5-13-0-arm7-qr': 'Android 13.0 Pixel5 Arm7',
-  'android-hw-p5-13-0': 'Android 13.0 Pixel5 Arm7',
+  'android-hw-p5-13-0-arm7': 'Android 13.0 Pixel5 Arm7',
```

Comment:
Dropping the old variant removes the nice name for tasks which used. Feel free to keep if it's used on beta or similar.

Reply:
I don't believe we use the old name anywhere- I am mostly trying to fix the new `wrench` jobs